test/enc_dec_plist.c

```diff
@@ -191,9 +191,6 @@ main(void)
                      "Testing ENCODE/DECODE with file version bounds: (%s, %s):", low_string, high_string);
             puts(msg);
 
-            if (VERBOSE_MED)
```

Comment:
This test never used the testframe.c framework and never changed the test verbosity, so checking for `VERBOSE_MED` had no effect.